Extensions/Terraform/Src/vss-extension.json

```diff
@@ -0,0 +1,178 @@
+{
+    "manifestVersion": 1,
+    "id": "custom-terraform-tasks",
+    "name": "Azure Devops Terraform integration",
+    "version": "0.0.1",
+    "publisher": "ms-vscs-rm",
+    "targets": [
+        {
+            "id": "Microsoft.VisualStudio.Services"
+        }
+    ],
+    "description": "Azure Devops extension for Terraform integration",
+    "categories": [
+        "Azure Pipelines"
+    ],
+    "icons": {
+      "default": "images/icon.png",
+      "large": "images/icon.png"
+    },
+    "files": [
+        {
+            "path": "Tasks/TerraformTaskV1"
+        },
+        {
+            "path": "Tasks/TerraformInstaller"
+        },
+        {
+            "path": "images",
+            "addressable": true
+        }
+    ],
+    "contributions": [
+        {
+            "id": "custom-terraform-installer-task",
+            "type": "ms.vss-distributed-task.task",
+            "targets": [
+                "ms.vss-distributed-task.tasks"
+            ],
+            "properties": {
+                "name": "Tasks/TerraformInstaller"
+            }
+        },
+        {
+            "id": "custom-terraform-release-task",
+            "type": "ms.vss-distributed-task.task",
+            "targets": [
+                "ms.vss-distributed-task.tasks"
+            ],
+            "properties": {
+                "name": "Tasks/TerraformTaskV1"
+            }
+        },
+        {
+            "id": "google-cloud-service-endpoint",
+            "description": "Credentials for connecting to Google Cloud Platform",
+            "type": "ms.vss-endpoint.service-endpoint-type",
+            "targets": [
+                "ms.vss-endpoint.endpoint-types"
+            ],
+            "properties": {
+                "name": "GoogleCloudServiceEndpoint",
+                "displayName": "GCP for Terraform",
+                "helpMarkDown": "[Get JSON Key File](https://console.cloud.google.com/iam-admin/serviceaccounts)",
+                "url": {
+                    "displayName": "Server Url",
+                    "helpText": "GCP homepage",
+                    "value": "https://www.googleapis.com/",
+                    "isVisible": "false"
+                },
+                "dataSources": [
+                    {
+                        "name": "GCP",
+                        "endpointUrl": "{{endpoint.url}}storage/v1/b?project=bold-vial-242405",
```

Comment:
Thanks a lot for pointing this out. The "project" request parameter is the project id of the GCP project in which the resources will be managed. That is an input parameter from the service endpoint. I have made the change and it will be included in the next iteration.